airflow.cfg

```diff
@@ -828,7 +828,7 @@ worker_concurrency = 16
 # If autoscale option is available, worker_concurrency will be ignored.
 # http://docs.celeryproject.org/en/latest/reference/celery.bin.worker.html#cmdoption-celery-worker-autoscale
 # Example: worker_autoscale = 16,12
-# worker_autoscale =
+worker_autoscale = 4,1
```

Comment:
i do not have an informed opinion on number of concurrent workers, but 16 seemed like a lot. regardless of how many we want to roll with, this autoscale setting seems better so that we don't have idle workers?

Reply:
i read about this at https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#worker-autoscale, and this feels right to me. 4 max and 1 min feels like a good place to start. 👍

Reply:
fwiw, we have been using the plain old `SequentialExecutor` in our [v1 deployment](https://github.com/cityofaustin/atd-airflow/blob/a5853a8974c270d1249c56bd147bb86f14718e51/config/airflow.cfg#L53), which makes me optimistic that even just 4 workers might suffice.